apps/services/models.py

```diff
@@ -0,0 +1,192 @@
+        return f'{self.__class__.__name__}(user_request_id="{self.user_request_id}",' \
+               f'date_created="{self.date_created}", comment="{self.comment}",user_id="{self.user_id}", service_id="{self.service_id}",' \
+               f'faculty_id="{self.faculty_id}", university_id="{self.university_id}", status_id="{self.status_id}")'
+
+
+class Status(Base):
+    __tablename__ = "status"
+
+    status_id = Column(INTEGER, primary_key=True, nullable=False)
+    status_name = Column(VARCHAR(length=50), nullable=False)
+
+    user_request = relationship("UserRequest", back_populates="status")
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}(status_id="{self.status_id}",status_name="{self.status_name}")'
+
+
+class Requisites(Base):
+    __tablename__ = 'requisites'
+
+    requisites_id = Column(INTEGER, primary_key=True, nullable=False)
+    iban = Column(VARCHAR(length=100))
+    organisation_code = Column(VARCHAR(length=50))
+    payment_recognition = Column(VARCHAR(length=255))
+    university_id = Column(INTEGER, ForeignKey("university.university_id"), nullable=False)
+    service_id = Column(INTEGER, ForeignKey("service.service_id"), nullable=False)
+
+    university = relationship("University", back_populates="requisites")
+    service = relationship("Service", back_populates="requisites")
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}(requisites_id="{self.requisites_id}",iban="{self.iban}",organisation_code="{self.organisation_code}",' \
+               f'payment_recognition="{self.payment_recognation}",university_id="{self.university_id}",service_id="{self.service_id}")'
+
+
+class UserRequestReview(Base):
+    __tablename__ = "user_request_review"
+    user_request_review_id = Column(INTEGER, primary_key=True, nullable=False)
+    date_created = Column(DATETIME, nullable=False)
+    room_number = Column(INTEGER)
+    start_date_accommodation = Column(DATETIME)
+    end_date_accommodation = Column(DATETIME)
+    total_sum = Column(FLOAT)
+    payment_deadline = Column(DATETIME)
+    remark = Column(VARCHAR(length=255))
+    date_review = Column(DATETIME, nullable=False)
+    bed_place_id = Column(INTEGER, ForeignKey("bed_place.bed_place_id"))
+    reviewer = Column(INTEGER, ForeignKey("user.user_id"), nullable=False)
+    hostel_id = Column(INTEGER, ForeignKey("hostel.hostel_id"))
+    university_id = Column(INTEGER, ForeignKey("university.university_id"), nullable=False)
+    user_request_id = Column(INTEGER, ForeignKey("user_request.user_request_id"), nullable=False)
+
+    bed_places = relationship("BedPlace", back_populates='user_request_reviews')
+    reviewer_user = relationship("User", back_populates='user_request_reviews')
+    hostels = relationship("Hostel", back_populates='user_request_reviews')
+    university = relationship("University", back_populates='user_request_reviews')
+    user_request = relationship("UserRequest", back_populates='user_request_reviews')
```

Comment:
The UserRequest has only one UserRequestReview,
so rename to "user_request_reviews" to `user_request_review`.